src/types/responses/Extrinsic.ts

```diff
@@ -29,3 +30,11 @@ export interface ISignature {
 	signature: EcdsaSignature | Ed25519Signature | Sr25519Signature;
 	signer: Address;
 }
+
+export interface IExtrinsicIndex {
+	at: {
```

Comment:
Should be able to use `IAt` interface here. Because of some legacy code height is a string, so will need to unwrap if its a compact and then `.toString(10)` to ensure it serializes as a base 10.

Reply:
Thanks for pointing that out. The beauty of `unwrap`